docs_guide/sphinx_guide/how_to_add_sections.rst

```diff
@@ -0,0 +1,95 @@
+####################################
+How to add sections to documentation
+####################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to separate your documents in parts. 
+
+.. include:: ../others/general_pre-requisites.rst
+
+You can organize your documents into:
+
+* Sections.
+* Subsections.
+* Subsubsections.
+* Paragraphs.
+
+Create sections
+===============
+
+Section headers are created by underlining (and optionally overlining) the section title with a punctuation character, at least as long as the text:
+
+.. code-block:: text
+
+    Section name
+    ============
+
+And it would look like this:
+
+Section name
+============
+
+Create subsections
+==================
+
+You can add a subsection to your section by writing the corresponding name covered below by a sign. This time it's the hyphen, ``-``. So you can include, for example, this subsection header:
+
+.. code-block:: text
+
+    Subsection name
+    ---------------
```

Comment:
Here we should change the wording too the one used for Sphinx's docs like you did for sections.